Example/PrimerSDK/Info.plist

```diff
@@ -15,7 +15,7 @@
 	<key>CFBundlePackageType</key>
 	<string>APPL</string>
 	<key>CFBundleShortVersionString</key>
-	<string>$(MARKETING_VERSION)</string>
+	<string>1.6</string>
```

Comment:
Why did this get hardcoded?

Reply:
So we're getting an issue later when building. I'm gonna see if I can reproduce.